core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/edit-ema-editor.component.html

```diff
@@ -2,6 +2,7 @@
     <dot-uve-toolbar />
 } @else {
     <dot-edit-ema-toolbar (editUrlContentMap)="editContentMap($event)" />
+    <!-- <dot-edit-ema-toolbar (changePageLanguage)="onChangePageLanguage($event)" /> -->
```

Comment:
A flag comment so you remember to clean this. Resolve it when we are going to merge 🙌🏽

Reply:
Thanks! That's helpful

Reply:
I'll create a ticket for this. 
Currently, we have two dialogs, and the one from Shell handles only two actions. We can update the Dialog component for UVE to manage all its actions through the store or service.